src/Maatwebsite/Excel/Classes/LaravelExcelWorksheet.php

```diff
@@ -27,7 +27,7 @@ class LaravelExcelWorksheet extends PHPExcel_Worksheet {
      * Parent
      * @var PHPExcel
      */
-    public $_parent;
+    private $_parent;
```

Comment:
I agree that they should be private, but we can't change that right now, as people might depend on them to be publically accessable. Can you revert this public -> private changes, please.